server/opendp_apps/terms_of_access/models.py

```diff
@@ -21,6 +33,30 @@ class Meta:
     def __str__(self):
         return f'{self.name} - v{self.version}'
 
+    def save(self, *args, **kwargs):
+        """
+        Save a TermsOfAccess model.
+        :param args:
+        :param kwargs: Potentially contains template_path
+        :return:
+        """
+        # For testing and flexibility, we should allow this method to accept an argument `template_path` which
+        # specifies the location of the template to be compared against.
+        try:
```

Comment:
- Saving the TOA through the Django admin fails. (e.g. how do you specify the `template_path` kwargs?)

Reply:
- e.g. What's the workflow for adding a file?

Reply:
The `template_path` field is really so that we can pass different templates in when testing. In production use, this should be hardwired to look in the file's current directory for `templates/` and then a html file with the same name as the version number. 

In order to save this via admin, do the following:

1.  Give it any name
2. Paste the contents of 1.0.html into the description field
3. Set the version to 1 (this tells the save function to look for a file called 1.0.html to do the comparison against)

Reply:
Looks good. Read the code wrong.